include.js

```diff
@@ -1766,6 +1766,13 @@ let cards = [
     author: 'Gaurav Saxena',
     githubLink: 'https://github.com/Gauravsaxena816'
   },
+  {
+    artName: 'Rocket Launch',
+    pageLink: './Art/dalvishruti14/index.html',
+    imageLink: './Art/dalvishruti14/rocketlaunch.gif',
```

Comment:
Have you checked if this works?

Your file name is `RocketLaunch.gif` (camel case), but in your entry you wrote `rocketlaunch.gif` (all lower case).

Unix systems (and the pipeline runs on Ubuntu) distinguish between cases.

So I suggest that you double-check or adjust your link to the `.gif`.